packages/backend-modules/search/lib/options.js

```diff
@@ -0,0 +1,106 @@
+const debug = require('debug')('search:lib:options')
+
+const { DOCUMENTS_SAMPLES_REPO_IDS, DOCUMENTS_SAMPLES } = process.env
+
+const processors = {
+  formatFeedSamples: (args, context) => {
+    const { filter } = args
+    const { user } = context
+
+    if (
+      DOCUMENTS_SAMPLES_REPO_IDS &&
+      !user?.roles?.includes('member') &&
+      filter?.format &&
+      DOCUMENTS_SAMPLES_REPO_IDS.split(',').includes(filter?.format)
```

Comment:
```suggestion
      !user?.roles?.includes('member') &&
      DOCUMENTS_SAMPLES_REPO_IDS?.split(',').includes(filter?.format)
```

but should we always use the helper `userIsInRole`? And maybe do the `DOCUMENTS_SAMPLES_REPO_IDS` split just once outside of the `formatFeedSamples` process function?

Reply:
Yes, `userIsInRole` is muches better and splitting ENV var outside fn call, too. Incorporated.